src/compiler/core.ts

```diff
@@ -534,30 +537,6 @@ export function* mapDefinedIterator<T, U>(iter: Iterable<T>, mapFn: (x: T) => U
     }
 }
 
-/** @internal */
-export function mapDefinedEntries<K1, V1, K2, V2>(map: ReadonlyMap<K1, V1>, f: (key: K1, value: V1) => readonly [K2, V2] | undefined): Map<K2, V2>;

```

Comment:
Any reason to delete this unused func but not other unused ones? Not that it much matters.

Reply:
What other unused functions are there? I saw that it only has a single use-site and could easily be substituted with `new Set(mapDefinedIterator(...))` where it was used.

Reply:
`compose` is unused IIRC. It has gone away and come back though.